index.js

```diff
@@ -121,6 +123,9 @@ Toolkit.run(async tools => {
     const remoteRepo = `https://${process.env.GITHUB_ACTOR}:${process.env.GITHUB_TOKEN}@github.com/${process.env.GITHUB_REPOSITORY}.git`
     if (process.env['INPUT_SKIP-TAG'] !== 'true') {
       await tools.runInWorkspace('git', ['tag', newVersion])
```

Comment:
now you add 2 tags? do you want an if-else here?

Reply:
No it updates the commit message on the specified tag

See https://stackoverflow.com/questions/7813194/how-do-i-edit-an-existing-tag-message-in-git/23532519

Reply:
ah ok thats cool, didnt know that exists 🙂

Reply:
I've just had a thought about an edge case where if only the `tag-commit-message` input is specified then any subscribed downstream workflows would be triggered twice as the original commit message does not include [skip-ci] by default.

Perhaps reverting to a simpler if/else would be better to address this problem. What do you think?

Reply:
ok, but thats only for devs that use it this way.

no existing repos would break right?